internal/cmd/iam/command_pool.go

```diff
@@ -4,6 +4,7 @@ import (
 	"github.com/spf13/cobra"
 
 	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	pconv "github.com/confluentinc/cli/internal/pkg/name-conversions"
```

Comment:
No reason to rename the package name here, there's no naming conflict as far as I can tell.
```suggestion
	"github.com/confluentinc/cli/internal/pkg/name-conversions"
```